tests/release/pipelines/push_to_external_registry.go

```diff
@@ -0,0 +1,196 @@
+			"roleVerbs":     {"get", "list", "watch"},
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = fw.AsKubeAdmin.CommonController.CreateRoleBinding("role-release-service-account-binding", managedNamespace, "ServiceAccount", releasecommon.ReleasePipelineServiceAccountDefault, managedNamespace, "Role", "role-release-service-account", "rbac.authorization.k8s.io")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	AfterAll(func() {
+		if !CurrentSpecReport().Failed() {
+			Expect(fw.AsKubeAdmin.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+			Expect(fw.SandboxController.DeleteUserSignup(fw.UserName)).To(BeTrue())
+		}
+	})
+
+	var _ = Describe("Post-release verification", func() {
+		It("verifies that a build PipelineRun is created in dev namespace and succeeds", func() {
+			Expect(fw.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component, "",
+				fw.AsKubeAdmin.TektonController, &has.RetryOptions{Retries: 2, Always: true})).To(Succeed())
+		})
+
+		It("verifies that a Release CR should have been created in the dev namespace", func() {
+			Eventually(func() error {
+				releaseCR, err = fw.AsKubeAdmin.ReleaseController.GetFirstReleaseInNamespace(devNamespace)
+				return err
+			}, releasecommon.ReleaseCreationTimeout, releasecommon.DefaultInterval).Should(Succeed())
+		})
+
+		It("verifies that Release PipelineRun is triggered", func() {
+			Eventually(func() error {
+				pr, err := fw.AsKubeAdmin.ReleaseController.GetPipelineRunInNamespace(managedNamespace, releaseCR.GetName(), releaseCR.GetNamespace())
+				if err != nil {
+					GinkgoWriter.Printf("release pipelineRun for release '%s' in namespace '%s' not created yet: %+v\n", releaseCR.GetName(), releaseCR.GetNamespace(), err)
+					return err
+				}
+				if !pr.HasStarted() {
+					return fmt.Errorf("pipelinerun %s/%s hasn't started yet", pr.GetNamespace(), pr.GetName())
+				}
+				return nil
+			}, releasecommon.ReleasePipelineRunCreationTimeout, releasecommon.DefaultInterval).Should(Succeed(), fmt.Sprintf("timed out waiting for a pipelinerun to start for a release %s/%s", releaseCR.GetName(), releaseCR.GetNamespace()))
+		})
+
+		It("verifies that Release PipelineRun should eventually succeed", func() {
+			Eventually(func() error {
+				pr, err := fw.AsKubeAdmin.ReleaseController.GetPipelineRunInNamespace(managedNamespace, releaseCR.GetName(), releaseCR.GetNamespace())
+				Expect(err).ShouldNot(HaveOccurred())
+				if !pr.IsDone() {
+					return fmt.Errorf("release pipelinerun %s/%s did not finish yet", pr.GetNamespace(), pr.GetName())
+				}
+				Expect(tekton.HasPipelineRunSucceeded(pr)).To(BeTrue(), fmt.Sprintf("release pipelinerun %s/%s did not succeed", pr.GetNamespace(), pr.GetName()))
+				return nil
+			}, releasecommon.ReleasePipelineRunCompletionTimeout, releasecommon.DefaultInterval).Should(Succeed())
+		})
+
+		It("verifies that Enterprise Contract Task has succeeded in the Release PipelineRun", func() {
```

Comment:
I am quite sure that if the release pipelineRun succeeded (the previous check), then this must have succeeded too. It may be a better post release plr validation test to check if an image was pushed to the quay repo

Reply:
In EC task, it checks if the image is valid and pullable in the beginning.

Reply:
Are the images in snapshot changed in `push-snapshot`, if not, the images has been checked in 'verify-enterprise-contract'.

Reply:
We can add a existing check by using 'skopeo' cmd if we can get the image URL.

Reply:
Replied on slack

Reply:
If/when the check is added, can you add to the Checkpoints for this test in the README like `The image from the snapshot is pushed to Quay`?

Reply:
The value of `releasedImagePushRepo` is a repo URL, not an image URL. Do we just check if the repo exist?

Reply:
We have no `.images` section in the RPA, so that means the image will be pushed to `latest` (if we added an images section to the RPA data field, this would change, so this is probably worth mentioning in a comment in the test file). You can get the digest like
`skopeo inspect --no-tags --format '{{.Digest}}' docker://quay.io/redhat-appstudio-qe/dcmetromap:latest`
The same could be run on the `containerImage` field of the component in the snapshot. They should match if the push was successful